config_ipa.go

```diff
@@ -35,7 +35,7 @@ import (
 // EmptyCodeHashPoint is a cached point that is used to represent an empty code hash.
 // This value is initialized once in GetConfig().
 var (
-	EmptyCodeHashPoint           *Point
+	EmptyCodeHashPoint           Point
```

Comment:
See the below comment to understand more about this.
But I changed this to be sure this kind of bug won't happen again, and we copy global values and not pointers which is safer.